lib/process-services-cloud/src/lib/form/components/form-cloud.component.spec.ts

```diff
@@ -974,6 +972,10 @@ describe('FormCloudComponent', () => {
 
         formComponent.form = new FormModel();
```

Comment:
the form should be instantiated through the detect change not manually, same for the outcome

Reply:
In this case does it really make sense? we are checking that the event is emitted.